api/DOMMatrixReadOnly.json

```diff
@@ -450,9 +452,56 @@
           }
         }
       },
+      "fromMatrix": {
+        "__compat": {
+          "description": "<code>fromMatrix()</code> static function",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMMatrixReadOnly/fromMatrix",
+          "support": {
+            "chrome": {
+              "version_added": "57"
+            },
+            "chrome_android": {
+              "version_added": "57"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "33"
+            },
+            "firefox_android": {
+              "version_added": "33"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "44"
+            },
+            "opera_android": {
+              "version_added": "45"
+            },
+            "safari": {
+              "version_added": "11"
+            },
+            "safari_ios": {
+              "version_added": "11.3"
+            },
+            "webview_android": {
+              "version_added": "57"
```

Comment:
```suggestion
              "version_added": "61"
```

Reply:
Ah-ha! That may explain several bits of confusion.